src/RequestHeaders.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Yiisoft\RequestProvider;
+
+final class RequestHeaders {
```

Comment:
```suggestion
final class RequestHeaderProvider {
```

Reply:
Suggested name is not consistent with `RequestCookies`.

Reply:
`RequestCookies` is inconsistent with the rest Yii3 packages as well

Reply:
Agree. `RequestHeaderProvider` is better name.

Reply:
Maybe a RequestHeaderCollection?

Reply:
Also good, but it's not a classic collection. Very custom implementation that depends on another class. It's more proxy than collection.
ProxyCollection? 😁

Reply:
In the end, what decision did you make?

Reply:
Let's call it `RequestHeaderProvider`